lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -1412,6 +1562,14 @@ def main():
             target_group_arns=dict(type='list'),
             availability_zones=dict(type='list'),
             launch_config_name=dict(type='str'),
+            launch_template=dict(type='dict',
+                                 default=None,
+                                 options=dict(
+                                     version=dict(type='str', required=True),
```

Comment:
Should version be required? Maybe it could default to the latest version of the launch template?

Reply:
I was in two minds about this; I wasn't sure wether it was better to make that assumption or just to require Version each time.  Possibly does make sense thinking about it now to have the version assume the default version unless it's provided.  Will update.